var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -344,16 +369,21 @@ def mpi_dependent_options(self):
                 '--with-cc=%s' % os.environ['CC'],
                 '--with-cxx=%s' % (os.environ['CXX']
                                    if self.compiler.cxx is not None else '0'),
-                '--with-fc=%s' % (os.environ['FC']
-                                  if self.compiler.fc is not None else '0'),
                 '--with-mpi=0'
             ]
+            if '+fortran' in self.spec and self.compiler.fc is not None:
```

Comment:
Shouldn't this error out if `+fortran` and there is no Fortran compiler available?

Reply:
Addressed.